.gitignore

```diff
@@ -1,4 +1,30 @@
-.idea/
+*.class
+*.idea
+.settings
+.project
+*.sass-cache
+target
+.java-version
+*.releaseBackup
+release.properties
+README.html
+
+# Package Files #
+*.jar
+*.war
+*.ear
 *.iml
-target/
-**\.DS_Store
+
+.settings/*
+api/.classpath
+api/.project
+api/.settings/*
+api/target/*
+omod/.classpath
+omod/.project
+omod/.settings/*
+omod/target/*
```

Comment:
**target** [line 6](https://github.com/Bahmni/bahmni-offline-sync/pull/1/files#diff-a084b794bc0759e7a6b77810e01874f2R6) takes care of this one

Reply:
Sorry I haven't checked in details the file, indeed there is duplicates. I have copied that from another module actually. I will remove the '.target' of line 6.

Reply:
I feel it's better to remove line 22, 26 instead `.target` of line 6

Reply:
Sure